hashes/enigma_machine.py

```diff
@@ -41,8 +41,7 @@ def engine(input_character):
 
 
 if __name__ == "__main__":
-    decode = input("Type your message:\n")
-    decode = list(decode)
+    decode = list(input("Type your message:\n"))
```

Comment:
Does this deal gracefully with the situation where the user enters the empty string?

Reply:
In its current form, an empty input results in an empty code. Alternatives would be to throw an error or to ask the user again.